{"name": "test_negative_delete_task_by_id", "status": "passed", "description": "Generate random task ID and try to delete it without permission, validate that received error message and code equal to expected", "steps": [{"name": "Delete task by ID: 6, expected status code: 401", "status": "passed", "steps": [{"name": "Create url with base url and endpoint: /task/6\nResult URL: https://api.clickup.com/api/v2/task/6", "status": "passed", "start": 1746889616727, "stop": 1746889616727}, {"name": "Send DELETE request to https://api.clickup.com/api/v2/task/6 with json=None and data=None, expected status code 401", "status": "passed", "start": 1746889616727, "stop": 1746889618153}], "start": 1746889616727, "stop": 1746889618153}], "start": 1746889616727, "stop": 1746889618153, "uuid": "475750b5-1508-433a-92e9-89fca0307783", "historyId": "111aa04c46da628e59bae88064c9bbd8", "testCaseId": "111aa04c46da628e59bae88064c9bbd8", "fullName": "tests.api.test_tasks.TestTasks#test_negative_delete_task_by_id", "labels": [{"name": "feature", "value": "Manage tasks"}, {"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_tasks"}, {"name": "subSuite", "value": "TestTasks"}, {"name": "host", "value": "apachai"}, {"name": "thread", "value": "11548-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_tasks"}]}